fashion-example-query/flows/query.yml

```diff
@@ -0,0 +1,34 @@
+!Flow
+version: '1'
+with:
+  read_only: true  # better add this in the query time
+  compress_hwm: 1024
+pods:
+  - name: encode
+    uses: pods/encode.yml
+    parallel: $JINA_PARALLEL
+  - name: indexer_tshirt
+    uses: pods/indexer-tshirt.yml
+    shards: $JINA_SHARDS
+    needs: [encode]
+    polling: all
+    uses_after: pods/reduce.yml
```

Comment:
make sure `reduce.yml` do proper sorting of results

Reply:
But this is the same as before no? It was/is already giving good results

Reply:
Yes I checked and is good